db/migrate/20171112174323_remove_polymorphism_from_ratings.rb

```diff
@@ -0,0 +1,32 @@
+class RemovePolymorphismFromRatings < ActiveRecord::Migration[5.1]
+  class Rating < ActiveRecord::Base; end
```

Comment:
Does this need a `self.table_name = 'ratings'`? Because of this being an inline class I'm thinking the table name would be auto-resolved to `remove_polymorphisms_from_ratings_ratings`

Reply:
Hm, good point! This is actually a scenario I never thought of before - but sounds plausible 🤔 

The reason why I thought this wouldn't be a problem: the migration-class is not an `ActiveRecord::Base` class and therefore does _(at least that's how I understand it)_ not add a prefix to its nested classes...

But what you say sounds plausible - I'll add it just to play save ✌️

Reply:
Just for reference @carpodaster: you were so right and I by coincidence also ran over a (or the 🤔 ) blog post about it today👇 
http://blog.testdouble.com/posts/2014-11-04-healthy-migration-habits.html

Reply:
> The ~ring~ article _wants_ to be found!

– Gandalf the Grey on Active Record migration techniques